base/src/com/google/idea/blaze/base/async/process/ExternalTask.java

```diff
@@ -61,12 +61,12 @@ default Future<Integer> runAsync(BlazeScope... scopes) {
 
   /** A builder for an external task */
   class Builder {
-    @VisibleForTesting public final ImmutableList.Builder<String> command = ImmutableList.builder();
```

Comment:
Why is this annotation being removed? If this field no longer needs to be visible for testing, its visibility should be changed too.